tests/ui/autoref-autoderef/issue-38940.rs

```diff
@@ -1,52 +0,0 @@
-// issue-38940: error printed twice for deref recursion limit exceeded
```

Comment:
I deleted this test because it is identical to [test/ui/did_you_mean/recusion_limit_deref.rs](https://github.com/rust-lang/rust/blob/master/tests/ui/did_you_mean/recursion_limit.rs) is exactly the same